### ISSUE TITLE
is there anyone local who could explain noflo to me?

### ISSUE THREAD

**Issue by govpack** [BrisJS/meetups#53]
## is there anyone local who could explain noflo to me?
i'm going to be looking into it for the purposes of re-using it's layout engine

so there's a brisJS social coming up, thurs april 23rd , but i may be too busy washing my hair, or writing my new book "test cases for head cases the 3rd edition" under my nom-de-plume "Eric Von Lipton the 3rd -- aka the resultinator" ~ in which we'll find out if that cat is doot or not, or has had kittens, or is well and TRUELY oot to lunch or off in la la land. hopefully everyone will give it the big tl;dr, or the too pricey not so nicey tp;nsn (yeah it's only going to cost you, how many hours and seconds?) or the old Jerry (Seinfeld)  serioulszly?! well they think they are, that's the funny part http://comediansincarsgettingcoffee.com

# tl;dr is just disastrous, 
#####  i mean i get the point :!) and i share the sentiment!! 
*_but if you take that attitude then you've stopped exploring because unfortunately we're stuck in README land (fighting the text wars) for a bit longer yet, not quite geological time but it feels that slow, like writing comments -- it sucks_* 

                   
1. noflo                     Flow-based programming for JavaScript
2. noflo-yaml                YAML handling components for NoFlo
3. noflo-mq                  Message Queue components for NoFlo
4. noflo-basecamp            BaseCamp Classic components for NoFlo
5. noflo-xml                 XML handling components for NoFlo
6. noflo-webserver           Web server components for NoFlo
7. noflo-html                HTML handling components for NoFlo
8. noflo-couchdb             CouchDB components for NoFlo
9. noflo-restfulmetrics      Restful Metrics components for NoFlo
10. noflo-ui-server           DEPRECATED, see README
11. noflo-liquid              Liquid Templating components for the NoFlo flow-based programming environment
12. noflo-markdown            Markdown processing components for NoFlo
13. noflo-diffbot             Diffbot components for NoFlo
14. noflo-filesystem          Filesystem components for NoFlo
15. noflo-github              GitHub components for NoFlo
16. noflo-git                 Git components for NoFlo
17. noflo-oembed              oEmbed components for NoFlo
18. noflo-redis               Redis components for NoFlo
19. noflo-graphviz            NoFlo visualization tools for GraphViz
20. noflo-ducksboard          Ducksboard components for NoFlo
21. noflo-test                Fluent test framework for NoFlo components
22. noflo-routers             Routing Packets in NoFlo
23. noflo-groups              Group Utilities for NoFlo
24. noflo-strings             String Utilities for NoFlo
25. noflo-objects             Object Utilities for NoFlo
26. noflo-flow                Flow Control for NoFlo
27. noflo-packets             Packet Utilities for NoFlo
28. noflo-adapters            Packet Format Conversion for NoFlo
29. grunt-init-noflo          grunt-init template for scaffolding NoFlo projects
30. noflo-dom                 Document Object Model components for NoFlo
31. noflo-math                Mathematical operations for NoFlo
32. polymer-noflo             Polymer Web Components wrapper for NoFlo
33. fbp                       FBP flow definition language parser
34. dataflow-noflo            Archived prototype. To see the current work:
35. noflo.github.io           The NoFlo website
36. component-fetcher         Utility for fetching the public list of NoFlo components and component libraries
37. noflo-docco               Parse source code into literate documentation
38. noflo-core                Core components for NoFlo
39. noflo-physics             Physics components for NoFlo
40. noflo-gravatar            Gravatar components for NoFlo
41. noflo-css                 Cascading Style Sheets components for NoFlo
42. noflo-interaction         User interaction components for NoFlo
43. noflo-ardrone             ARDrone components for NoFlo
44. noflo-localstorage        HTML5 LocalStorage components for NoFlo
45. noflo-websocket           WebSocket components for NoFlo
46. noflo-runtime-iframe      IFRAME runtime for NoFlo
47. noflo-ui                  NoFlo Development Environment
48. noflo-ajax                AJAX components for NoFlo
49. noflo-runtime-base        Base library for building NoFlo runtimes
50. noflo-runtime-websocket   NoFlo runtime for execution on Node.js over WebSockets
51. noflo-gestures            Gesture recognition library for NoFlo
52. noflo-polymer             Polymer components for NoFlo
53. noflo-indexeddb           IndexedDB components for NoFlo
54. noflo-knex                Knex.js database query components for NoFlo
55. noflo-blockchain          blockchain.info components for NoFlo
56. noflo-mandrill            Mandrill email API components for NoFlo
57. visualize                 Simple FBP visualizer
58. noflo-nodejs              Node.js runtime environment for NoFlo UI
59. noflo-graph               Graph manipulation components for NoFlo
60. noflo-runtime             FBP runtime components for NoFlo
61. noflo-finitedomain        Finite Domain Constraint Solving for NoFlo
62. noflo-image               Image processing components for NoFlo.
63. noflo-embedly             Embed.ly API components for NoFlo
64. noflo-tween               
65. noflo-facebook            Facebook components for NoFlo
66. noflo-canvas              Generative Canvas 2D drawing with NoFlo.
67. grunt-noflo-manifest      Grunt plugin for updating NoFlo manifest files
68. component-fbp             Component plugin for requiring FBP files
69. noflo-gnome               NoFlo runtime for GNOME development
70. grunt-noflo-browser       Grunt plugin for easier browser builds of NoFlo
71. noflo-video               Video handling components for NoFlo
72. noflo-ccv                 ccv face detection for noflo
73. noflo-amd                 Asynchronous Module Definition components for NoFlo
74. noflo-react               Facebook React components for NoFlo
75. noflo-amqp                Advanced Message Queuing Protocol components for NoFlo
76. noflo-thegrid             TheGrid components for NoFlo
77. noflo-google              Google API components for NoFlo
78. noflo-helper-arrayable    (WIP)
79. noflo-peer                NoFlo wrapper for peers/peerjs P2P audio, video, and data via WebRTC.
80. noflo-tika                Document extraction components for NoFlo
81. noflo-s3                  S3 components for NoFlo
82. noflo-runtime-webrtc      WebRTC runtime transport
83. noflo-browser-app         Template for noflo-browser applications
84. noflo-mirobot             Components for Mirobot.io
85. noflo-browserfile         Browser file input (and drop) for NoFlo.
86. noflo-color               Color manipulation for NoFlo.
87. noflo-webaudio            Web Audio API for NoFlo.
88. noflo-geometry            Components for computational geometry.
89. noflo-mqtt                MQTT components for NoFlo
90. noflo-gpio                GPIO components for Embedded Linux (RPi/BBB etc)
91. noflo-imgflo              Components for imgflo.
92. noflo-runtime-msgflo      NoFlo runtime for AMQP/MQTT
93. noflo-svg                 (WIP)
94. noflo-stanbol             Apache Stanbol components for NoFlo.
95. noflo-sharp               Fast image resizing components for NoFlo.

**Comment by kevdesign** [comment]
I actually kickstarter backed the noflo project and it's node support... flow based programming is pretty concise and seems to tie into functional programming concepts well. I need to revisit it, so i can't really comment on it yet - but we should check it out sometime
